lib/common.js

```diff
@@ -939,6 +939,7 @@ module.exports = {
             }
 
             var _opts = {
+                log: req.log,
```

Comment:
Correct me if I'm wrong: all we are doing with `libmantalite.normalizeMantaPath` usage is setting `req.key`?
And req.key is used exactly once here: https://github.com/joyent/manta-buckets-api/blob/3b9fa981dff336159989f16bed0789149ed8b8b8/lib/buckets/list.js#L21  in a log statement. IIUC it is a relic of *moray* usage where a moray object has a "key" field as its unique key.  Now we have a SQL "PRIMARY KEY" (https://github.com/joyent/manta-buckets-mdapi/blob/master/schema_templates/schema.in#L52).

If true, I think the normalization func and usage can be dropped.

Reply:
so, hm. you are absolutely right.  I thought I remembered this key specifically in the listing logic, but i'm remembering `r.key` where `r` is from `mreq.on('record', ...)`.... and even then that was swapped for `r.name`.

I just checked on my lab machine inside `/opt/smartdc/buckets-api` to ensure that node_modules was checked as well, and I don't think `req.key` is used in any capacity.  I'll drop it and do some testing, good find!